pkg/burner/create.go

```diff
@@ -123,18 +128,29 @@ func (ex *Executor) RunCreateJob(iterationStart, iterationEnd int) {
 		}
 		log.Debugf("Creating object replicas from iteration %d", i)
 		if ex.Config.NamespacedIterations {
```

Comment:
Can we rewrite this logic as below, please?
```suggestion
		if ex.Config.NamespacedIterations {
                        nsIndex := i / ex.Config.IterationsPerNamespace
                        ns := fmt.Sprintf("%s-%d", ex.Config.Namespace, nsIndex)
                        if namespacesCreated[ns] {
                            continue
                        }
                        if err := createNamespace(ClientSet, ns, nsLabels); err != nil {
                            log.Error(err.Error())
                            continue
                        }
                        namespacesCreated[ns] = true
               }

```